modules/git-flow-hooks-config.sh.dist

```diff
@@ -3,3 +3,5 @@ VERSION_BUMPLEVEL_HOTFIX="PATCH"
 VERSION_BUMPLEVEL_RELEASE="MINOR"
 VERSION_SORT="/usr/bin/sort -V"
 VERSION_TAG_PLACEHOLDER="%tag%"
+USE_CHANGELOG=false
+CHANGELOG_FILE_NAME="CHANGELOG"
```

Comment:
I'd prefer a common prefix to these variables, in this case `CHANGELOG_`.
Also, `CHANGELOG_FILE` would be more in line with `VERSION_FILE`.

Could you change the settings to:

```sh
CHANGELOG_ENABLE=false
CHANGELOG_FILE="CHANGELOG"
```